OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,10 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Altitude where this bullet should explode when reached.",
+			"Negative values allow this bullet to pass cliffs and terrain bumps.")]
+		public readonly WDist ExplodeUnderThisAltitude = WDist.Zero;
```

Comment:
My real (but less diplomatic) reasoning is that I don't think we should explicitly support hacks that work around lazyness-induced bugs.  Implementing this "officially" as an airbust parameter means that mods can use it for things like cluster artillery, and you can still use it how you want without us having to spend hours arguing over it.